common/src/main/java/com/microsoft/identity/common/internal/providers/oauth2/AuthorizationResultFactory.java

```diff
@@ -0,0 +1,28 @@
+package com.microsoft.identity.common.internal.providers.oauth2;
+
+import android.content.Intent;
+
+/**
+ * Abstract Factory class to construct {@link AuthorizationResult}
+ */
+
+public abstract class AuthorizationResultFactory {
```

Comment:
😄

Reply:
Haha irony !. Edited to make it a bit more logical.